examples/new_design_examples/new_design_msv.m

```diff
@@ -0,0 +1,98 @@
+clear all
+close all
+import casadi.*
+import vdx.*
+
+R = 3.5;
+%% Define projected system
+problem_options = nosnoc.Options();
+solver_options = nosnoc.solver.Options();
+%problem_options.rk_scheme = RKSchemes.GAUSS_LEGENDRE;
+%problem_options.rk_scheme = RKSchemes.LOBATTO_IIIC;
+problem_options.rk_scheme = RKSchemes.RADAU_IIA;
+%problem_options.rk_representation= RKRepresentation.differential_lift_x; 
```

Comment:
this option leads to strage results in my case.

Reply:
I believe this is fixed. At least on my end it seems to work correctly.